fedbiomed/node/requests/_n2n_controller.py

```diff
@@ -122,22 +129,33 @@ async def _HandlerKeyRequest(self, overlay_msg: dict, inner_msg: InnerMessage) -
         Returns:
             A `dict` with overlay reply message
         """
-        # TODO: replace with real payload, waiting for DH key pair to be generated,
-        # then send it.
-        import random
-        # Use 15 seconds delay to have sometimes success, sometimes failure
-        # delay = random.randrange(1, 15)
-        delay = random.randrange(1, 12)
-        for i in range(delay):
-            logger.debug(f"===== WAIT 1 SECOND IN NODE TO NODE ROUTER {i+1}/{delay}")
-            await asyncio.sleep(1)
 
+        # # TODO: replace with real payload, waiting for DH key pair to be generated,
+        # # then send it.
+        # import random
+        # # Use 15 seconds delay to have sometimes success, sometimes failure
+        # # delay = random.randrange(1, 15)
+        # delay = random.randrange(1, 5)
+        # for i in range(delay):
+        #     logger.debug(f"===== WAIT 1 SECOND IN NODE TO NODE ROUTER {i+1}/{delay}")
+        #     await asyncio.sleep(1)
```

Comment:
Is this comment still relevant?

Reply:
(not existing in last committed version)